EssentialsDiscord/src/main/java/net/essentialsx/api/v2/services/discord/InteractionEvent.java

```diff
@@ -0,0 +1,59 @@
+package net.essentialsx.api.v2.services.discord;
+
+/**
+ * A class which provides information about what triggered an interaction event.
+ */
+public interface InteractionEvent {
+    /**
+     * Creates/Appends the initial response message.
+     * @param message The message to append.
+     */
+    void reply(String message);
+
+    /**
+     * Gets the member which caused this event.
+     * @return the member which caused the event.
+     */
+    InteractionMember getMember();
+
+    /**
+     * Helper method to get the String representation of the argument by the given key or null if none by that key is present.
```

Comment:
```suggestion
     * Get the value of the argument matching the given key represented as a String, or null if no argument by that name is present. 
```

Ditto for other methods